App.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Text, View} from 'react-native';
 
 if (__DEV__) {
-  module.exports = require("./target/scala-2.12/app-fastopt.js").app;
+  module.exports = require("./target/scala-2.13/scalajs-bundler/main/app-fastopt.js").app;
```

Comment:
Generally, for React Native applications we don't need to use Webpack since React Native comes with its own bundler. Is there a specific reason you made this change?

Reply:
I understand, this is removed.